apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -287,6 +303,9 @@ export default function RubricsContainer({
           size={Button.ButtonSize.narrow}
         />
       </div>
+      <div style={styles.bottomRow}>
```

Comment:
Could you also disable the save button when saving?

Reply:
[Done! ](https://github.com/code-dot-org/code-dot-org/pull/53727/files#r1324854664)